presto-hive/src/main/java/io/prestosql/plugin/hive/HiveTableProperties.java

```diff
@@ -161,6 +165,28 @@ public static String getAvroSchemaUrl(Map<String, Object> tableProperties)
         return Optional.ofNullable((Integer) tableProperties.get(TEXTFILE_SKIP_FOOTER_LINE_COUNT));
     }
 
+    public static Optional<Character> getTextFieldSeparator(Map<String, Object> tableProperties)
+    {
+        return getTextFormatProperty(tableProperties, TEXTFILE_FIELD_SEPARATOR);
+    }
+
+    public static Optional<Character> getTextFieldSeparatorEscape(Map<String, Object> tableProperties)
+    {
+        return getTextFormatProperty(tableProperties, TEXTFILE_FIELD_SEPARATOR_ESCAPE);
+    }
+
+    private static Optional<Character> getTextFormatProperty(Map<String, Object> tableProperties, String key)
```

Comment:
`getTextFormat*Separator*Property` ?

also, isn't this equivalent to, but implemented differently than, `getCsvSerdeProperty`?